tests/installation/select_patterns_and_packages.pm

```diff
@@ -60,6 +60,11 @@ sub gotopatterns {
         send_key 'alt-s';
     }
     else {
+        # See poo#12322. Prevent tab presses before overview is fully loaded
+        if (!check_screen('installation-settings-overview-loaded')) {
```

Comment:
looks strange ... we just detected that we failed to find "Installation Summary" screen when we expecting to have it and only thing that we doing about that is just record info screen ? Are you sure there is a point to do send_key_until_needlematch after that ?